src/components/Review/DecisionArea.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react'
+import { Button, Header, Modal, Radio, Segment, TextArea, TextAreaProps } from 'semantic-ui-react'
+import { SummaryViewWrapper } from '../../formElementPlugins'
+import { ReviewResponseDecision } from '../../utils/generated/graphql'
+import { DecisionAreaState, ReviewQuestionDecision } from '../../utils/types'
+import strings from '../../utils/constants'
+
+interface DecisionAreaProps {
+  state: DecisionAreaState
+  setDecision: Function
+  submitHandler: (_: any) => void
+  problemMessage: string
+  setProblemMessage: Function
+}
+
+const DecisionArea: React.FC<DecisionAreaProps> = ({
+  state,
+  setDecision,
+  submitHandler,
+  problemMessage,
+  setProblemMessage,
+}) => {
+  const { open, review, summaryViewProps } = state
+  const handleChange = (value: ReviewResponseDecision) => {
+    const { id, comment } = review as ReviewQuestionDecision
+    setProblemMessage('')
+    setDecision({
+      ...state,
+      review: { id, comment, decision: value },
+    })
+  }
+
+  const handleUpdateComment = (_: any, { value }: TextAreaProps) => {
+    const { review } = state
+    setDecision({ ...state, review: { ...review, comment: value } })
+  }
+
+  return (
+    <Modal
+      closeIcon
+      open={open}
+      onClose={() => setDecision({ open: false })}
+      size="fullscreen"
+      style={{ margin: 0, background: 'transparent' }}
+    >
+      {open && summaryViewProps && review && (
+        <Segment
+          floated="right"
+          style={{
+            backgroundColor: 'white',
+            height: '100vh',
+            width: '300px',
+          }}
+        >
+          <Segment basic>
+            <Header>{strings.TITLE_DETAILS}</Header>
+            <SummaryViewWrapper {...summaryViewProps} />
```

Comment:
This is the generic SummaryViewWrapper, right? This might not be appropriate for elements that have a custom SummaryViewWrapper. I guess we need to find a way to import the specific component using the manifest? Maybe create a new issue for this, as it's not important right now, but will be needed at some point.

Reply:
I'm not sure if I understood the problem. Wouldn't the Wrapper call the specific SummaryView based on the type?

Reply:
Oh sorry, yes you're right. I was thinking of the SummaryVIew, but you've got the SummaryViewWrapper. All good :)